### PR TITLE
Updating Attacks: Credential Stuffing

### PR DESCRIPTION
**Pull request by nickmalcolm**
This is in response to the request for contributions put out on OWASP Slack. This has hopefully clearer language, verifiable & better referenced claims, and a clearer diagram.

1) Follow the likes of the XSS page in structure
2) Remove some verbosity and jargon to hopefully make it more accessible to people
3) Update references with newer sources
4) Update the diagram to actually show credential stuffing
5) Add references to other OWASP pages (brute forcing & password spraying)

### CODE REVIEW

File: pages/attacks/Credential_stuffing.md [hunk 1]
```diff
@@ -13,45 +13,43 @@ tags: attack, Credential stuffing
...
-3.  Successful logins (usually 0.1-0.2% of the total login attempts) allow the attacker to take over the account matching the stolen credentials.
-4.  The attacker drains stolen accounts of stored value, credit card numbers, and other personally identifiable information
-5.  The attacker may also use account information going forward for other nefarious purposes (for example, to send spam or create further transactions)
+1.  The attacker acquires usernames and passwords from a website breach, phishing attack, password dump site.
+2.  The attacker uses automated tools to test the stolen credentials against many websites (for instance, social media sites, online marketplaces, or web apps).
+3.  If the login is successful, the attacker knows they have a set of valid credentials.
+
+Now the attacker knows they have access to an account. Potential next steps include:
+
+1.  Draining stolen accounts of stored value or making purchases
```
Comment (kingthorin) [inline pages/attacks/Credential_stuffing.md:38]:
```suggestion
1.  Draining stolen accounts of stored value or making purchases.
```

File: pages/attacks/Credential_stuffing.md [hunk 2]
```diff
@@ -13,45 +13,43 @@ tags: attack, Credential stuffing
...
+2.  Accessing sensitive information such as credit card numbers, private messages or pictures, or documents
```
Comment (kingthorin) [inline pages/attacks/Credential_stuffing.md:39]:
```suggestion
2.  Accessing sensitive information such as credit card numbers, private messages, pictures, or documents.
```

File: pages/attacks/Credential_stuffing.md [hunk 3]
```diff
@@ -13,45 +13,43 @@ tags: attack, Credential stuffing
...
+3.  Using the account to send phishing messages or spam
```
Comment (kingthorin) [inline pages/attacks/Credential_stuffing.md:40]:
```suggestion
3.  Using the account to send phishing messages or spam.
```

File: pages/attacks/Credential_stuffing.md [hunk 4]
```diff
@@ -13,45 +13,43 @@ tags: attack, Credential stuffing
...
+4.  Selling known-valid credentials to one or more of the compromised sites for other attackers to use
```
Comment (kingthorin) [inline pages/attacks/Credential_stuffing.md:41]:
```suggestion
4.  Selling known-valid credentials to one or more of the compromised sites for other attackers to use.
```

**Review by kingthorin — CHANGES_REQUESTED** [review]
Thanks for tackling this, just a few minor things.

**Review by kingthorin — APPROVED** [review]
Thanks!

**Comment by nickmalcolm** [comment]
No worries! Good team work! 🙌🏻

**Pull request merged**